core/files/assets/svg/svg-handler.php

```diff
@@ -336,7 +336,19 @@ private function get_allowed_attributes() {
 			'textlength',
 		];
 
-		return apply_filters( 'elementor/files/svg/allowed_attributes', $allowed_attributes );
+		/**
+		 * Allowed attributes in SVG file.
```

Comment:
Since the SVG can run JS code that may inject malicious code - all attributes are removed except the allowed attributes. please consider before adding new attributes.